VERSION.txt

```diff
@@ -140,11 +181,13 @@ jetty-9.4.39.v20210325 - 25 March 2021
  + 6052 Cleanup TypeUtil and ModuleLocation to allow jetty-client/hybrid to
    work on Android
  + 6063 Allow override of hazelcast version when using module
- + 6072 jetty server high CPU when client send data length > 17408 - Resolves CVE-2021-28165
+ + 6072 jetty server high CPU when client send data length > 17408 - Resolves
+   CVE-2021-28165
  + 6085 Jetty keeps Sessions in use after "Duplicate valid session cookies"
    Message
  + 6101 Normalize ambiguous URIs - Resolves CVE-2021-28164
- + 6102 Exclude webapps directory from deployment scan - Resolves CVE-2021-28163
+ + 6102 Exclude webapps directory from deployment scan - Resolves
+   CVE-2021-28163
 
 jetty-9.4.38.v20210224 - 24 February 2021
```

Comment:
Doesn't this file also need 9.4.41 to be included?

Reply:
yup fixed